src/book/03-guides/04-general/01-git.mdx

```diff
@@ -64,6 +66,28 @@ If your code is in a different repository and it uses C++ or protobuf files, use
 
 If you push your changes to GitHub and get a message that the build failed due to formatting issues, try running the relevant format commands, commit any changes, and push to GitHub again. This will trigger a new build, which will check the formatting again. You won't be able to merge the PR until the build checks pass.
 
+## Reviewing Code
+
+To review code on GitHub open up a PR and open the Files Changed tab. This will give you the diff of the PR. You can click on the lines of code in the diff to add a comment and/or a suggestion to those lines. You can either leave a single comment or start a review. In a review all comments you make will have pending written on them to say that they have not been posted yet. Please make each comment non-ambiguous and clearly state what your concern is.
+
+When reading the code, if there is anything that confuses you or think is incorrect make a comment with all your concerns. For example, if there is a nontrivial block of code without a comment explaining it you should ask for the author to provide such a comment. Another example is if there is a block of code that could do something that the autor may have not considered you should inform the author, with a comment, on what these other effects may be.
+
+When finished reading the code and you have a bunch of pending comments for your review you can do one of three things. The first is to request changes, this blocks the code from being merged until you approve the branch, even if others have approved it. The second is to approve the PR, this allows it to be merged into master, don't do this unless you are confident that the code is correct. The third is to leave a comment, do this if you are not too concerened about the comments you have made.
+
+Reviewing code that you may not be fully confident with and leaving comments on helps both yourself learn and the author make more readable code.
+
+## Writing Reviewable Code and Reviewee Etiquette
+
+Writting reviewable code is an art, here are some guidelines that you should stick to to make it easier for reviews:
+
+- **Atomicity**: A reviewable PR should only address one issue or add one feature.
+- **Length**: A reviewable PR should not change much more than 400 lines of code, anything above this super-exponentioally increases the time it takes to review.
+- **Name**: A reviewable PR should have a verb saying what you did the the issue fixed or the feature added.
```

Comment:
```suggestion
- **Name it well**: A reviewable PR should be named starting with a verb saying what you did, what issue was fixed, or what feature was added. Prefer the present tense, e.g. "Update dependency X to version Y" instead of "Updated dependency X to version Y".
```